src/components/nano/NanoContractHistory.js

```diff
@@ -132,7 +132,6 @@ function NanoContractHistory({ ncId }) {
             // so we must clear the query params
             pagination.clearOptionalQueryParams();
           }
-          return;
```

Comment:
Why was this removed?

Reply:
This was removed because of the [`no-useless-return`](https://eslint.org/docs/latest/rules/no-useless-return) lint rule.